pkg/network/state.go

```diff
@@ -518,6 +521,9 @@ func (ns *networkState) mergeConnections(id string, active map[string]*Connectio
 		ns.createStatsForKey(client, key)
 		ns.updateConnWithStats(client, key, c)
 
+		if c.Last.IsZero() {
```

Comment:
Does this need to be done before calling `createStatsForKey` or `updateConnWithStats`?  Does it matter?